### ISSUE TITLE
Getting Warning Error 001839

### ISSUE THREAD

**Issue by slouque** [Esri/collector-tools#87]
I am getting this error telling me

a datum transformation is expected, but the parameter is empty

![Warning 001839](https://user-images.githubusercontent.com/62031277/81605232-c6ecb300-9396-11ea-879f-399c8b977ea4.png)

 Can someone decifer why this is happening.

 I ran the tool earlier with no problem

**Comment by Kevin-GIS** [comment]
Hi @slouque,

For the Output Coordinate System, when you previously ran this successfully did you also choose a VCS of `NAVD88_height_`?

I reproduced the issue as well when using those same spatial references. However it worked successfully when using NAVD88 in meters. 

The reason for this issue is likely due to the unit variations with NAVD88_height being in feet and NAVD88 being in meters. When determining a suitable datum transformation, oftentimes when any unit conversations take place, due to the underlying complexities involved in that scenario, it can cause the datum transformations to be unavailable due to the amount of individual transformations that need to take place to handle that conversion. 

You could try using NAVD88 and then running the normal Project GP tool to convert to NAVD88(Height)(ftus) for the VCS. Or you could simply keep the output in meters and use the field calculator in the attribute table to convert the Z values to feet in a new column.

Thanks
-Kevin

**Comment by slouque** [comment]
Thanks for that info Kevin.

I think you are right. I was so confused why I had got it to work earlier and now all of a sudden I was getting an error.

Scott Louque
St James Parish GIS Analyst
225-562-2431

**Comment by Kevin-GIS** [comment]
Thank you @slouque.

Please let me know if you run into any further issues with the Project Z tool.

I am going to close this one out.

Thanks
-Kevin

**Comment by slouque** [comment]
Thank you Kevin!

I went out and collected some points this morning and it worked great

**Comment by Kevin-GIS** [comment]
That's awesome @slouque 

Out of curiosity:
- What type of GPS receiver are you working with?
- What OS are you running Collector with?
- What type of features are you capturing in the field?
- How many total field users are working with Collector?

Thank you for that info.
-Kevin

**Comment by slouque** [comment]
We have a Trimble R2 GNSS receiver
I have an ipad with software version 13.3.1 on it
We are going to be using it to capture elevation values of for our hydrology
So far I am the only one that has used the receiver with Collector. We are supposed to have two Utility workers who will be using it to collect the points but they haven’t come get it yet.

That’s all I know so far.

Thanks
Scott Louque
St James Parish GIS Analyst
225-562-2431

**Comment by Kevin-GIS** [comment]
Thank you for sharing that information with me about your workflow @slouque!

Please let me know if you run into any issues with the tool.

Thank you
-Kevin

**Comment by slouque** [comment]
Kevin,

Yesterday morning I took the receiver out to where one of our elevation monuments is located and collected a point. Came back into the office and ran the Project Z tool on the feature class containing the point I went collect. Plus the Project tool and Add XY Coordinates tool which the tutorial I followed said. It gave me a value very close to what the last calibrated reading for that monument was back in 2013. The reading on this monument from 2013 was 19.39 ft and the reading I got yesterday morning was 19.92 ft

Yesterday evening I went out again to the same monument to collect another point which I ran through the same process (I think) and I got a totally different elevation value than what I got in the morning. 25 ft

So this morning I went to another area to try again and am still getting a different reading than what I initially got.

My steps are as follows.

First I open up my web map in ArcGIS Pro and export the hosted feature class out to a file geodatabase feature class
Then I bring that feature class into the Project Z tool

[cid:image001.png@01D629CD.4F5F04B0]

Then once get that value I use the Project tool to project into a coordinate system we use here and then transform it into a NAVD88 (height)(USft)

[cid:image002.png@01D629CD.C31E84C0]

Once I have an elevation value, I then use the Add XY Coordinates tool to convert it to ft

[cid:image003.png@01D629CE.60174D70]

Am I missing a step?

I had it working what seemed perfect before

Note: Sorry for such a long email and let me know if you want me to add this to the github forum

Thanks,

Scott Louque
St James Parish GIS Analyst
225-562-2431
